src/Lachain.Storage/Trie/TrieHashMap.cs

```diff
@@ -132,16 +135,23 @@ private IEnumerable<byte[]> TraverseValues(ulong root)
                     break;
                 default:
                     foreach (var child in node.Children)
-                    foreach (var value in TraverseValues(child))
-                        yield return value;
+                        foreach (var value in TraverseValues(child))
```

Comment:
It is better to not commit occasional changes like this and several other changes in this file. It makes search in commit history more difficult in the future